rust-runtime/aws-smithy-http-server/src/routing/mod.rs

```diff
@@ -62,6 +62,10 @@ pub use self::{future::RouterFuture, into_make_service::IntoMakeService, route::
 /// [awsJson1.1]: https://awslabs.github.io/smithy/1.0/spec/aws/aws-json-1_1-protocol.html
 /// [endpoint trait]: https://awslabs.github.io/smithy/1.0/spec/core/endpoint-traits.html#endpoint-trait
 #[derive(Debug)]
+#[deprecated(
+    since = "0.52.0",
+    note = "OperationRegistry is part of the deprecated service builder API. This type no longer appears in the public API."
```

Comment:
Do we want to provide a link to some actionable documentation on how to use the new builder API in the deprecation message?

Reply:
Good idea. There's a few more [documents from the book](https://github.com/awslabs/smithy-rs/blob/main/design/src/server/code_generation.md) I've got to update still - I'll then link to them?

Reply:
Or perhaps it'd be better to link to a stand alone migration guide?

Reply:
A standalone migration guide would be great, but the book would be good enough as well if we have a dedicated section.